backend/User/validators.py

```diff
@@ -5,7 +5,7 @@
 def validate_image(file):
 	valid_mime_types = ['image/jpeg', 'image/jpg', 'image/png']
 	valid_file_extensions = ['.jpg', '.jpeg', '.png']
-	max_file_size = 2097152  # 2MB
+	max_file_size = 1048576
```

Comment:
is this fixed from the frontend too?

Reply:
and settings.py

Reply:
does the frontend doesn't check and just sends to the backend whatever it gets